subsys/canbus/canopen/canopen_sync.c

```diff
@@ -47,4 +47,4 @@ static void canopen_sync_thread(void *p1, void *p2, void *p3)
 
 K_THREAD_DEFINE(canopen_sync, CONFIG_CANOPEN_SYNC_THREAD_STACK_SIZE,
 		canopen_sync_thread, NULL, NULL, NULL,
-		CONFIG_CANOPEN_SYNC_THREAD_PRIORITY, 0, K_MSEC(1));
+		CONFIG_CANOPEN_SYNC_THREAD_PRIORITY, 0, 1);
```

Comment:
Not related to this PR or your script, but why this discrepancy between `K_THREAD_DEFINE()` and `k_thread_create()`? Shouldn't the type of the last parameter to `K_THREAD_DEFINE()` mirror the last parameter to `k_thread_create()`? (i.e. be `k_timeout_t`). @andyross comments?

Reply:
Following up to myself: I got the answer on Slack: K_MSEC() resolves to a function call now, so it cannot be used in static context.